soccer_vision_3d_msgs/package.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>soccer_vision_3d_msgs</name>
+  <version>0.0.0</version>
+  <description>A package containing some vision related message definitions in the soccer domain.</description>
```

Comment:
Maybe we include a note on 3-dimensionality in there. (Probably also applies to other package.xml files)

Reply:
Added